### ISSUE TITLE
failed to use SKLEARNEX for 'pairwise_distances'

### ISSUE THREAD

**Issue by lbjcom** [intel/scikit-learn-intelex#716]
patching `pairwise_distances` does not work when I test with following code:
```python
from sklearnex import patch_sklearn
patch_sklearn()
import numpy
from sklearn.metrics import pairwise_distances
from sklearn.cluster import KMeans
import pdb

x = numpy.random.rand(10000,10000)
dist = pairwise_distances(x)

kmeans_clusterer = KMeans(n_clusters=10,
                          init="k-means++",
                          max_iter=300,
                          random_state=0)

labels = kmeans_clusterer.fit_predict(x)
```
Other functions (kmeans) work properly.

### Reproduce
- Run above code with `export SKLEARNEX_VERBOSE=INFO`
- Output:
```bash
Intel(R) Extension for Scikit-learn* enabled (https://github.com/intel/scikit-learn-intelex)
SKLEARNEX INFO: sklearn.metrics.pairwise_distances: fallback to original Scikit-learn
SKLEARNEX INFO: sklearn.cluster.KMeans.fit: running accelerated version on CPU
```

### Expected Behavior
`SKLEARNEX INFO: sklearn.metrics.pairwise_distances: fallback to original Scikit-learn` should not be printed.

### Environment
 - OS: Ubuntu 18.04
 - python 3.6.9
    - numpy 1.19.5
    - scikit-learn 0.24.2
    - scikit-learn-intelex 2021.3.0

**Comment by PetrovKP** [comment]
Yes, that's to be expected. Currently, the `pairwise_distances` support only "cosine" and "correlation" metrics for Intel(R) Extension for Scikit-learn, full list pathed algorithms, see [here](https://intel.github.io/scikit-learn-intelex/algorithms.html).  For `pairwise_distances` default is "euclidean" metric, which is currently don't patched.

**Comment by lbjcom** [comment]
> Yes, that's to be expected. Currently, the `pairwise_distances` support only "cosine" and "correlation" metrics for Intel(R) Extension for Scikit-learn, full list pathed algorithms, see [here](https://intel.github.io/scikit-learn-intelex/algorithms.html). For `pairwise_distances` default is "euclidean" metric, which is currently don't patched.

Thank you for the reply. I've seen the list but just checked the last column 😭
